modules/dreamview/frontend/src/renderer/ground.js

```diff
@@ -23,20 +23,71 @@ export default class Ground {
             return false;
         }
 
-        const { xres, yres, mpp } = PARAMETERS.ground;
+        if (!this.render(coordinates)) {
+            return false;
+        }
+
+        this.initialized = true;
+        return true;
+    }
+
+    update(world, coordinates, scene) {
+    }
+
+    updateImage(coordinates, mapName) {
+        if (!this.mesh) {
+            return;
+        }
+        const dir = this.titleCaseToSnakeCase(mapName);
+        /*import(`assets/map_data/${dir}/background.jpg`)
```

Comment:
is it okay to remove commented code?

Reply:
Surely ok to remove. Was trying to leave a working code sample for dynamic import